src/browser/wpe/test/ocdm_client.cpp

```diff
@@ -69,7 +69,7 @@ int ocdmClient::streamUpdate(void* ocdmDrmStream, unsigned char* pbResponse, int
     return ret;
   }
 
-  ret = g_pOpenCdm->Update(pbResponse, cbResponse, responseMsg);
+  ret = g_pOpenCdm->Update(pbResponse, cbResponse, responseMsg) == OpenCdm::KeyStatus::Usable ? 0 : 1;
```

Comment:
If it's usable ret is false? That seems wrong, but I've seen this weird switching of truth values in the code elsewhere, but I try to fix them when I find it.

Reply:
I didn't try to fix it as I don't think it is the moment to fix everything I get into because I could begin and non-stop for weeks, so I tried to keep the answer consistent and the thing is that if the key is usable it means that there is no message, which means returning 0 if you want to keep the same behavior as before, which I did.